libs/CloudscalerLibcloud/CloudscalerLibcloud/utils/network/rules.py

```diff
@@ -22,6 +22,7 @@
 CLEANUPFLOWS_CMD_IP = '''\
 ovs-ofctl del-flows {bridge} "ip,nw_src={ipaddress}";
 ovs-ofctl del-flows {bridge} "ip,nw_dst={ipaddress}";
+ovs-ofctl del-flows {bridge" "arp,nw_src={ipaddress}";
```

Comment:
Is the syntax of this line correct?

looks like the bridge is not closed correctly with }, but instead a " is used